rest/config.go

```diff
@@ -130,6 +130,7 @@ type DbConfig struct {
 	Unsupported          db.UnsupportedOptions          `json:"unsupported,omitempty"`                 // Config for unsupported features
 	OIDCConfig           *auth.OIDCOptions              `json:"oidc,omitempty"`                        // Config properties for OpenID Connect authentication
 	ViewQueryTimeoutSecs *uint32                        `json:"view_query_timeout_secs,omitempty"`     // The view query timeout in seconds
+	localDocExpirySecs   *uint32                        `json:"local_doc_expiry_secs,omitempty"`       //The _local doc expiry time in seconds
```

Comment:
This should be exported (LocalDocExpirySecs), so that it's included in things like /db/_config

Reply:
LocalDocExpirySecs has been exported